src/lib/input/autosize.ts

```diff
@@ -17,9 +17,25 @@ export class MdTextareaAutosize implements OnInit {
   /** Minimum number of rows for this textarea. */
   @Input() minRows: number;
 
+  get mdAutosizeMinRows(): number {
+    return this.minRows;
+  }
+
+  @Input() set mdAutosizeMinRows(value: number) {
+    this.minRows = value;
+  }
+
   /** Maximum number of rows for this textarea. */
   @Input() maxRows: number;
 
+  get mdAutosizeMaxRows(): number {
+    return this.maxRows;
+  }
+
+  @Input() set mdAutosizeMaxRows(value: number) {
```

Comment:
The test component in the spec still has `minRows` and `maxRows`. It should use the updated API.